python/dgl/core.py

```diff
@@ -184,7 +184,7 @@ def bucketor(data):
         return bkts
     return unique_val, bucketor
 
-def data_dict_to_tuple(graph, data_dict, op, lhs_list=None, rhs_list=None):
+def data_dict_to_list(graph, data_dict, func, target):
```

Comment:
Any reason changing it to a list?

Reply:
Made it easier to handle unary and binary functions. The tupling is now handled in ops/spmm.py and ops/sddmm.py